src/components/CommunityPartners/CommunityPartnersModal.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+  <transition leave-active-class="duration-300">
+    <div
+      v-show="isModalOpen"
+      class="fixed z-10 inset-0 overflow-y-auto"
+      role="dialog"
+    >
+      <div
+        class="
+          flex
+          items-end
+          justify-center
+          min-h-screen
+          pt-4
+          px-4
+          pb-20
+          text-center
+          sm:block sm:p-0
+        "
+      >
+        <transition name="ease-out-overlay">
+          <div
+            v-show="isModalOpen"
+            class="fixed inset-0 bg-gray-500 bg-opacity-75 transition-opacity"
+            @click="toggleModal"
+          ></div>
+        </transition>
+
+        <!-- This element is to trick the browser into centering the modal contents. -->
+        <span class="hidden sm:inline-block sm:align-middle sm:h-screen">&#8203;</span>
+
+        <transition name="ease-out-modal">
+          <!-- Modal Content -->
+          <div
+            v-show="isModalOpen"
+            class="
+              inline-block
+              align-middle
+              bg-white
+              rounded-lg
+              text-left
+              overflow-hidden
+              shadow-xl
+              transform
+              transition-all
+              w-full
+              md:w-9/12
+              2xl:w-7/12
+            "
+          >
+
+          <form class="dark:bg-gray-800" autocomplete="off" v-on:submit.prevent="handleFormSubmit">
+            <div class="flex">
+              <div class="text-white md:flex flex-col bg-indigo-700 dark:bg-indigo-800 w-1/3 hidden p-10 gap-7">
+              <p class="text-xl ">Contact Information</p>
+              <p class="text-lg">For more information, please reach out to our support team.</p>
```

Comment:
```suggestion
              <p class="text-lg">Have any questions/comments, or want more information? Contact any of our team members via <a href="https://discord.gg/VwJp4KM">Discord</a>.</p>
```

Reply:
I think this is better?
 'Have any questions/comments? Reach out to our team members via discord'

Reply:
Sounds good!

Reply:
Yeah, less word = better.

Reply:
Here's how it will look:
![image](https://user-images.githubusercontent.com/28738855/146868713-8f018db1-6026-4bdc-9156-265bf11a579c.png)

What do you guys think?

Reply:
looks good